config.yaml

```diff
@@ -80,3 +80,40 @@
 # Defaults to: "604800" (one week)
 
 # jwt_exp_timeout: 604800
+
+
+# Redis address
+# Defaults to: "", which disables the Redis cache
+# Overwrite with environment variable: DEVICEAUTH_REDIS_ADDR
+
+# redis_addr: ""
+
+# Redis username
+# Defaults to: "", which disables the Redis authentication
+# Overwrite with environment variable: DEVICEAUTH_REDIS_USERNAME
+
+# redis_username: ""
+
+# Redis password
+# Defaults to: "", which disables the Redis authentication
+# Overwrite with environment variable: DEVICEAUTH_REDIS_PASSWORD
+
+# redis_password: ""
+
+# Redis DB
+# Defaults to: 0
+# Overwrite with environment variable: DEVICEAUTH_REDIS_DB
+
+# redis_db: "0"
+
+# Redis client timeout (sec)
+# Defaults to: 5
+# Overwrite with environment variable: DEVICEAUTH_REDIS_TIMEOUT_SEC
+
+# redis_timeout_sec: "5"
```

Comment:
5 seconds is a rather high value, and it is going to kill the backend if Redis becomes unresponsive.
What do you think about lowering it to 1 second?

Reply:
don't mind it at all, I'll set it to 1 sec.

it makes sense since we're best effort and want to get out quickly in case of problems.